encrypted-media-respec.html

```diff
@@ -1606,22 +1606,26 @@
       <h2><a>MediaKeySession</a> Interface</h2>
       <p>The MediaKeySession object represents a <a href="#key-session">key session</a>.</p>
       <p>
-        The User Agent SHALL execute the <a def-id="monitor-cdm-algorithm"></a> algorithm continuously for each <a>MediaKeySession</a> object until the sooner of
-        the <a def-id="session-closed-algorithm"></a> algorithm being run or the destruction of the <a>MediaKeySession</a>.
+        A <a>MediaKeySession</a> object is <dfn id="media-key-session-closed">closed</dfn> if and only if the <a def-id="session-closed-algorithm"></a> algorithm has
+        been run.
+      </p>
+      <p>
+        The User Agent SHALL execute the <a def-id="monitor-cdm-algorithm"></a> algorithm continuously for each <a>MediaKeySession</a> object
+        that is accessible to the page and that is not <a def-id="media-key-session-closed"></a>.
```

Comment:
I'm not sure about the "accessible to the page" language. I'm not sure "destruction" is correct either (as it relates to garbage-collected objects), but I think we should stick with that until we get clarification otherwise.